.gitignore

```diff
@@ -29,4 +29,5 @@ hs_err_pid*
 #Eclipse auto-generated files
 .classpath
 .project
-*core.prefs
\ No newline at end of file
+*.core.prefs
+*.core.resources.prefs
```

Comment:
Can't you combine these two lines into `*.core.*`?

Reply:
I don't know if there are any other .core. files not related to Eclipse. Might instead change it to *.prefs .